### PR TITLE
VZ-6783: Parameterize Jenkins tests to choose CRD version. 

### PR DESCRIPTION
**Pull request by Abhijithang**
This PR sets the default install config for Verrazzano CRD API version to v1beta1 and provides a parameter to run custom tests across pipelines.

### CODE REVIEW

File: tests/e2e/config/scripts/v1beta1/install-verrazzano-olcne.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+apiVersion: install.verrazzano.io/v1beta1
+kind: Verrazzano
+metadata:
+  name: my-verrazzano
+spec:
+  components:
+    dns:
+      wildcard:
+        domain: nip.io
+    ingress:
```
Comment (anders-swanson) [inline tests/e2e/config/scripts/v1beta1/install-verrazzano-olcne.yaml:13]: ingressNGINX

File: JenkinsfileCopyrightTest [hunk 2]
```diff
@@ -52,7 +52,7 @@ pipeline {
         OCR_CREDS = credentials('ocr-pull-and-push-account')
         OCR_REPO = 'container-registry.oracle.com'
         IMAGE_PULL_SECRET = 'verrazzano-container-registry'
-        INSTALL_CONFIG_FILE_KIND = "./tests/e2e/config/scripts/install-verrazzano-kind.yaml"
+        INSTALL_CONFIG_FILE_KIND = "./tests/e2e/config/scripts/v1beta1/install-verrazzano-kind.yaml"
```
Comment (anders-swanson) [inline JenkinsfileCopyrightTest:55]: why is this one not parameterized?

**Review by anders-swanson — DISMISSED** [review]
LGTM

**Review by anders-swanson — APPROVED** [review]
LGTM

**Comment by anders-swanson** [comment]
You have conflicts, can you resolve these @Abhijithang

**Review by anders-swanson — DISMISSED** [review]

File: ci/multicluster/Jenkinsfile [hunk 3]
```diff
@@ -151,9 +155,9 @@ pipeline {
         OCI_CLI_REGION = "${params.OKE_CLUSTER_REGION}"
         OCI_CLI_SUPPRESS_FILE_PERMISSIONS_WARNING = 'True'
 
-        INSTALL_CONFIG_FILE_KIND = "${GO_REPO_PATH}/verrazzano/tests/e2e/config/scripts/install-vz-prod-kind-upgrade.yaml"
-        INSTALL_CONFIG_FILE_OCIDNS = "${GO_REPO_PATH}/verrazzano/tests/e2e/config/scripts/install-verrazzano-ocidns.yaml"
-        INSTALL_CONFIG_FILE_NIPIO = "${GO_REPO_PATH}/verrazzano/tests/e2e/config/scripts/install-verrazzano-nipio.yaml"
+        INSTALL_CONFIG_FILE_KIND = "${GO_REPO_PATH}/verrazzano/tests/e2e/config/scripts/v1beta1/install-vz-prod-kind-upgrade.yaml"
```
Comment (anajosep) [inline ci/multicluster/Jenkinsfile:158]: why is this path not parameterized ?

**Review by anajosep — APPROVED** [review]
LGTM, Please verify all the builds whose `Jenkinsfile` had been modified.

**Review by agv426 — APPROVED** [review]

File: ci/multicluster/Jenkinsfile (hunk 3 above)
Comment (Abhijithang) [reply ci/multicluster/Jenkinsfile:158]: When I tried to parameterize the env variables I am hitting the byte code limit reached issue with the jenkins build. Therefore I reverted to setting the config in the method.

Comment (agv426) [reply ci/multicluster/Jenkinsfile:158]: You can make more "space" in the generated bytecode by moving some of the initialization code into groovy functions. See the periodic test Jenkinsfile for an example pattern for doing that.

**Review by anders-swanson — DISMISSED** [review]

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**